packages/forms/src/components/Form/index.tsx

```diff
@@ -356,15 +356,10 @@ export default class Form<Data extends object = {}> extends React.Component<
 
     field.data.config = config;
     // @ts-ignore
-    field.initial = value;
-    // @ts-ignore
     field.value = value;
     field.touched = config.validateDefaultValue || false;
-
-    /* eslint-disable no-param-reassign */
-    formState.initialValues = setIn(formState.initialValues!, name, value);
```

Comment:
this might be a breaking change for apps that use initialValues?

Reply:
It still works (look a few lines above). This was just setting the value back to itself.

Reply:
This actually broke form resets.